cfg/pre-commit-config.yaml

```diff
@@ -1,6 +1,15 @@
 # cspell:words autofix, markdownlint, mdformat, shfmt
 ---
 repos:
+-
```

Comment:
Per eng call, update to cron job that runs once daily and creates an issue with any broken links